typescript/api-cors-lambda-crud-dynamodb/lambdas/create.ts

```diff
@@ -1,13 +1,15 @@
-const AWS = require('aws-sdk');
-const db = new AWS.DynamoDB.DocumentClient();
-const uuidv4 = require('uuid/v4');
+import * as AWS from 'aws-sdk';
```

Comment:
While it contradicts the above comment, I think non-specific imports for the AWS SDK makes sense even if it is contradictory since the SDK is specifically designed for use with general imports.